stdlib/public/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+
+include(CheckCXXCompilerFlag)
```

Comment:
Do you need this here if it's in SwiftSharedCMakeConfig.cmake?

Reply:
i.e. IWYU is good for stuff in cmake/, but all these little ones are assumed piles and piles of stuff has been set up anyway.

Reply:
If the inclusion is guaranteed then it is not strictly needed.  However, there are currently too many paths into the various files (hopefully we can start whittling away at some of that and simplify this).

Reply:
@jrose-apple - is that something you really want me to change or were you just curious?

Reply:
I'm mildly against applying IWYU to CMake files not in cmake/modules/, mostly because I'm mildly against them doing anything that requires includes in the first place.

Reply:
@jrose-apple - hmm, except the problem is that this really is a top level directory technically (its the top level for the runtime which is why we cannot really hoist the checks to the source root which is the top level for the tools.

Reply:
Then it should do something like when the overlays are treated as top level, and immediately include something in cmake/modules/.

Reply:
That seems reasonable.